diff --git a/nltk/stem/snowball.py b/nltk/stem/snowball.py
index b76b8a4..bd8e06e 100644
--- a/nltk/stem/snowball.py
+++ b/nltk/stem/snowball.py
@@ -824,7 +824,7 @@ class EnglishStemmer(_StandardStemmer):
                 break
 
         # STEP 1c
-        if word[-1] in u"yY" and word[-2] not in self.__vowels and len(word) > 2:
+        if len(word) > 2 and word[-1] in u"yY" and word[-2] not in self.__vowels:
             word = u"".join((word[:-1], u"i"))
             if len(r1) >= 1:
                 r1 = u"".join((r1[:-1], u"i"))
